### DISCUSSION TITLE
Can't write to RAW file format

### DISCUSSION THREAD

**Discussion by Kieranator** [ImageMagick/ImageMagick#7535] [Help]
### ImageMagick version

version 7, built from source, 64 bit with all extras enabled
Version: ImageMagick 7.1.1-37 (Beta) Q64-HDRI x64 c01fd3718:20240806

### Operating system, version and so on

Windows 11

### Description

https://imagemagick.org/script/formats.php

This page says that imagemagick can read and write RAW files, but when running magick identify -list format, the result for raw files is

RAW  DNG       r--   Raw (0.21.0-Beta1)

showing that it can only be read. In the source code, there's the IM_MOD_raw project, that appears to be full support for reading and writing raw images. So why isn't it properly being registered into imagemagick? 

Attempting to write a RAW image file yields this error: 

magick.exe: no encode delegate for this image format `RAW' @ warning/constitute.c/WriteImage/1400.

**Comment by dlemstra** [comment]
The `IM_MOD_raw` module is for "real" raw files (only Red/Blue/Green/etc.. channels). The `IM_MOD_dng` module is used for reading raw DNG files. There is no support for writing dng files. 

We should update our documentation to make this more clear. The `RAW` should be changed to `GRAY` and we should add `RAW` with a better description.

**Comment by Kieranator** [comment]
Ah, interesting! GRAY is actually the format that I want to be using for this application (high resolution heightmaps), however none of the applications I'm using can import it. I was specifically looking at RAW because it can be imported as a grayscale heightmap through Unreal Engine, which seems to be the only way to import a heightmap with greater than 16-bit precision. I'm not sure what image format Unreal Engine is actually looking for in those .raw files, or whether or not it can actually represent heightmaps higher than 16-bits internally.
